charts/datadog-crds/templates/datadoghq.com_datadogslos_v1.yaml

```diff
@@ -196,10 +249,4 @@ spec:
       storage: true
       subresources:
         status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
same

Reply:
same.. what I read when I spotted it was that this is added by API server anyway.